src/device/scrollCursorIntoView.ts

```diff
@@ -44,11 +69,19 @@ const scrollIntoViewIfNeeded = (el: Element | null | undefined) => {
   // scroll to 1 instead of 0
   // otherwise Mobile Safari scrolls to the top after MultiGesture
   // See: touchmove in MultiGesture.tsx
+  preventScrollDetection()
```

Comment:
Is `preventScrollDetection` intended to be called with every invocation of `window.scrollTo`, or only the ones in `scrollCursorIntoView`?

Reply:
Only the ones in `scrollCursorIntoView`. The idea here is to not cancel itself when called multiple times as thoughts are loaded.